modules/installation-vsphere-machines.adoc

```diff
@@ -154,33 +144,59 @@ specify values on the *Customize template* tab.
 +
 [IMPORTANT]
 ====
-If you plan to add more compute machines to your cluster after you finish
+If you plan to add more worker machines to your cluster after you finish
```

Comment:
I had thought the language was supposed to be `compute` and `control-plane`

Reply:
I couldn't remember which way we landed and noticed that we had both in this doc, so wanted to be consistent. I'll switch `worker` -> `compute`. Thanks!